contexts/retail-coupon/v1.json

```
@@ -0,0 +1,116 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "gs1": "https://gs1.org/voc/",
+
+    "ClippedCouponCredential": {
+      "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#ClippedCouponCredential"
+    },
+    "clippedCoupon": {
+      "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#clippedCoupon",
+      "@type": "@id"
+    },
+    "name": {
+      "@id": "https://schema.org/name"
+    },
+    "image": {
+      "@id": "https://schema.org/image"
+    },
+    "url": {
+      "@id": "https://schema.org/url"
+    },
+
+    "CouponOffer": {
+      "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#CouponOffer",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "gs1": "https://gs1.org/voc/",
+
+        "base_gs1": {
+          "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#base_gs1"
+        },
+        "brand_id": {
+          "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#brand_id"
+        },
+        "description": { "@id": "https://schema.org/description" },
+        "campaign_start_time": {
+          "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#campaign_start_time",
+          "@type": "xsd:dateTime"
```

`xsd` is not defined. Either define it somewhere or inline the full URL.  Similar for other uses here.
```suggestion
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
```

---

if using the full URL we should use `https` instead if possible.

---

In this case I think we need to use `http`.  This is a situation of an "identifier" URL that has been long and widely used as `http` and tools may depend on that.  w3.org has all the HTTP things setup to redir and force `https` if you dereference it.  There are some other `http` ones here: https://www.w3.org/TR/json-ld/#conformance.  But of course, schema.org is an exception where they historically and unfortunately started with `http` and now using `https`.

---

The official namespace they use is listed here: https://www.w3.org/TR/xmlschema-2/#namespaces